src/Frontend/src/components/messages/SequenceDiagram.vue

```diff
@@ -0,0 +1,46 @@
+<script setup lang="ts">
+import Endpoints from "./SequenceDiagram/EndpointsComponent.vue";
+import Timeline from "./SequenceDiagram/TimelineComponent.vue";
+import Handlers from "./SequenceDiagram/HandlersComponent.vue";
+import Routes from "./SequenceDiagram/RoutesComponent.vue";
+import { useSequenceDiagramStore } from "@/stores/SequenceDiagramStore";
+import { storeToRefs } from "pinia";
+
+const store = useSequenceDiagramStore();
+store.setConversationId("39907d51-12e5-4202-82c3-b2b30077ebd4");
+
+const { maxWidth, maxHeight } = storeToRefs(store);
+</script>
+
+<template>
+  <div class="outer">
+    <svg class="sequence-diagram" :width="`max(100%, ${isNaN(maxWidth) ? 0 : maxWidth}px)`" :height="maxHeight + 20">
```

Comment:
Should we be rendering at all if `isNaN(maxWidth)` is `true`?
I have created a [spinner](https://github.com/Particular/ServicePulse/pull/2311/files#diff-fd610f09e5530881446332f2c2ba6694b7fdbe268db643e96baed8f251fd3415) for exactly this purpose?